src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/AvoidUninstantiatedInternalClasses.cs

```diff
@@ -314,24 +314,27 @@ private static bool ContainsEntryPoint(INamedTypeSymbol type, Compilation compil
                 return false;
             }
 
+            var taskSymbol = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task");
```

Comment:
Please use [WellKnownTypes](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/WellKnownTypes.cs#L129)

Reply:
I didn't see Task in the list, but I  just overlooked. Is it an idea to add GenericTask to WellKnowTypes as well?